modules/client-configuration/pages/clients-ubuntu.adoc

```diff
@@ -74,12 +74,31 @@ The channels you need for this procedure are:
 | Security Channel
 | Client Channel
 
+<<<<<<< HEAD
```

Comment:
Please create a render of this documentation. I am quite sure this is wrong, as well as other lines later.

Seems you have an unfixed conflict.